hail/python/test/hail/experimental/test_experimental.py

```diff
@@ -1,5 +1,6 @@
 import hail as hl
 import unittest
+import numpy as np
```

Comment:
Is this used?

Reply:
Whoops forgot to clean that up nice catch.